### PR TITLE
add editing to plant card

### PR DESCRIPTION
**Pull request by gulfemakkaya**
Now you can edit two sections of a plant card

### CODE REVIEW

File: components/Form/index.js [hunk 1]
```diff
@@ -11,10 +11,7 @@ export default function Form() {
     event.preventDefault();
     const formData = new FormData(event.target);
     const newPlant = Object.fromEntries(formData);
-    // const newPlant = {
-    //   name: plantName,
-    //   wateringTime: wateringTime,
-    // };
+
     setPlants([...plants, { ...newPlant, id: crypto.randomUUID() }]);
     console.log(newPlant);
```
Comment (AGeck113) [inline components/Form/index.js:16]: please remove console.log

File: components/Form/index.js [hunk 2]
```diff
@@ -37,6 +34,14 @@ export default function Form() {
           </option>
           <option value="all two days">all two days</option>
         </select>
+        <label htmlFor="Place"> Place: </label>
+        <select name="place" id="Place">
+          <option value="living room" id="Place">
+            living room
+          </option>
```
Comment (AGeck113) [inline components/Form/index.js:41]: Maybe you could add 2-3 rooms more, to show, that it works. Should work as before with these options. 
```
 <option value="living room" id="Place">living room</option> 
<option value="kitchen" id="Place"> Kitchen  </option> 
<option value="basement" id="Place">Basement</option>
```
Edit: Maybe you have to do the same changes on your card

File: components/PlantCard/index.js [hunk 3]
```diff
@@ -2,32 +2,70 @@ import { createGlobalStyle } from "styled-components";
...
 export default function PlantCard() {
   const [plants, setPlants] = useAtom(globalPlants);
   function handleDelete(id) {
     setPlants(plants.filter((plant) => plant.id !== id));
   }
+  function handleSubmit(event, id) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    console.log(data);
```
Comment (AGeck113) [inline components/PlantCard/index.js:17]: remove console.log()

File: components/PlantCard/index.js [hunk 4]
```diff
@@ -2,32 +2,70 @@ import { createGlobalStyle } from "styled-components";
...
+                }}
+              >
+                <label htmlFor="Place"> Place: </label>
+                <select name="place" id="Place">
+                  <option selected value={plant.place}>
+                    {plant.place}
+                  </option>
+                  <option value="living room" id="Place">
+                    living room
+                  </option>
```
Comment (AGeck113) [inline components/PlantCard/index.js:50]: here could you add the options as well

File: components/TitleBar/index.js [hunk 5]
```diff
@@ -1,12 +1,22 @@
...
   left: 50%;
   transform: translateX(-50%);
   z-index: 100;
+  background-color: #c2d6c5;
+  width 100%
+  
+  
 `;
+
+const StyledTitleBar = styled.div``;
```
Comment (AGeck113) [inline components/TitleBar/index.js:22]: You don't give the div any Styling and I think you are not even using it

File: pages/_app.js [hunk 6]
```diff
@@ -16,3 +17,7 @@ export default function App({ Component, pageProps }) {
     </>
   );
 }
+
+// const StyledBody = styled.body`
+//   background-color: #;
+// `;
```
Comment (AGeck113) [inline pages/_app.js:23]: No outcommented code

File: components/PlantCard/index.js [hunk 7]
```diff
@@ -2,32 +2,70 @@ import { createGlobalStyle } from "styled-components";
...
+                </select>
+                <label htmlFor="Last watering"> Last watering: </label>
+                <input
+                  type="date"
+                  name="date"
+                  defaultValue={plant.date}
+                ></input>
+                <button type="submit">submit</button>
+              </form>
```
Comment (doemser) [inline components/PlantCard/index.js:59]: that this form is part of the plant card is a bit odd. i would expect that selecting the date would be enough to submit it. You could achieve this by controlling the inputs.

File: components/TitleBar/index.js [hunk 8]
```diff
@@ -1,12 +1,22 @@
 import styled from "styled-components";
 
 export default function TitleBar() {
-  return <H1Item>Planty</H1Item>;
+  return (
+    <div>
+      <H1Item>Planty</H1Item>
+    </div>
+  );
```
Comment (doemser) [inline components/TitleBar/index.js:8]:
```suggestion
  return <H1Item>Planty</H1Item>
```

File: components/TitleBar/index.js (hunk 5 above)
Comment (doemser) [inline components/TitleBar/index.js:22]: Is this unused?

**Review by doemser — CHANGES_REQUESTED** [review]
Actually, only small additions. However, you should adjust the structure with the form in the plant card, also that the add form is titled via "open popup" and "close popup" is very irritating.

File: components/PlantCard/index.js [hunk 9]
```diff
@@ -44,3 +96,101 @@ const StyledList = styled.ul`
...
+//   border-radius: 10px;
+//   margin: 10px;
+//   text-align: center;
+// `;
+
+// const StyledList = styled.ul`
+//   list-style-type: none;
+//   margin: 2px;
+//   padding: 2px;
+// `;
```
Comment (NikRoe) [inline components/PlantCard/index.js:196]: you should remove the code that is in that comment

```suggestion
```

File: components/PlantCard/index.js [hunk 10]
```diff
@@ -2,32 +2,84 @@ import { createGlobalStyle } from "styled-components";
 import styled from "styled-components";
 import Image from "next/image";
 import globalPlants from "@/public/data";
-
+import { atomWithStorage } from "jotai/utils";
```
Comment (NikRoe) [inline components/PlantCard/index.js:5]: are you using atomWithStorage here?

File: components/TitleBar/index.js [hunk 11]
```diff
@@ -1,12 +1,20 @@
...
 const H1Item = styled.h1`
   text-align: center;
   position: fixed;
   left: 50%;
   transform: translateX(-50%);
   z-index: 100;
+  background-color: #c2d6c5;
+  width 100%
+  
+
```
Comment (NikRoe) [inline components/TitleBar/index.js:19]:
```suggestion
```

File: public/data.js [hunk 12]
```diff
@@ -1,3 +1,4 @@
+import PlantCard from "@/components/PlantCard";
```
Comment (NikRoe) [inline public/data.js:1]: are you using this component here?

File: components/Form/index.js [hunk 13]
```diff
@@ -4,29 +4,31 @@ import { useState } from "react";
...
     setPlants([...plants, { ...newPlant, id: crypto.randomUUID() }]);
-    console.log(newPlant);
   }
 
   return (
     <>
       <h2>Add a new plant</h2>
-      <form type="add-edit" onSubmit={handleSubmit}>
+      <form
+        type="add-edit"
```
Comment (NikRoe) [inline components/Form/index.js:22]: what purpose does this typo serve? 🤔

Comment (gulfemakkaya) [reply components/Form/index.js]: I now realized that it had no use 😬 and changed your requests!

**Pull request merged**